tests/test_mujoco.py

```diff
@@ -0,0 +1,47 @@
+from stable_baselines.common.policies import MlpPolicy
+from stable_baselines.common.vec_env import DummyVecEnv
+from stable_baselines import PPO2
+
+import gym
+import benchmark_environments
+
+def test_fixed_env_model_as_good_as_gym_env_model():
+    train_timesteps = 200000
+    num_eval_episodes = 50
+    verbose = 0
+
+    gym_env = gym.make('HalfCheetah-v3')
+    fixed_env = gym.make('FixedHalfCheetah-v0')
+
+    gym_model = PPO2(MlpPolicy, gym_env, verbose=verbose)
+    gym_model.learn(total_timesteps=train_timesteps)
+
+    fixed_model = PPO2(MlpPolicy, fixed_env, verbose=verbose)
+    fixed_model.learn(total_timesteps=train_timesteps)
+
+    gym_reward = _eval_model(gym_model, gym_env, num_episodes=num_eval_episodes,
+            max_timesteps_per_episode=fixed_env._time_horizon)
+    fixed_reward = _eval_model(fixed_model, fixed_env, num_episodes=num_eval_episodes,
+            max_timesteps_per_episode=fixed_env._time_horizon)
+
+    epsilon = 0.1
+    sign = 1 if gym_reward > 0 else -1
+    assert (1 - sign * epsilon) * gym_reward <= fixed_reward
+
+def _eval_model(model, env, *, num_episodes, max_timesteps_per_episode):
```

Comment:
Does https://github.com/hill-a/stable-baselines/blob/master/stable_baselines/common/evaluation.py do what you need?